core/src/components/datetime/datetime.tsx

```diff
@@ -291,6 +297,22 @@ export class Datetime implements ComponentInterface {
    */
   @Watch('value')
   protected valueChanged() {
+    if (this.hasValue()) {
+      /**
+       * Update the value of the cloned reference to `activeParts` to synchronize
+       * the date display on the current render cycle.
+       */
+      const { month, day, year, hour, minute } = parseDate(this.value);
```

Comment:
Can we clarify this to explain how we want the selected values to change, but we do not want the user's calendar view to be forced to the selected month if they are looking at a different month?